src/Tasks/Microsoft.NET.Build.Extensions.Tasks/Microsoft.NET.Build.Extensions.Tasks.csproj

```diff
@@ -0,0 +1,75 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- This is a smaller build of Microsoft.NET.Build.Tasks to be used outside
+       of SDK projects -->
+
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <RootNamespace>Microsoft.NET.Build.Tasks</RootNamespace>
+    <TargetFrameworks>netcoreapp1.0;net46</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netcoreapp1.0</TargetFrameworks>
+    <GenerateDependencyFile>false</GenerateDependencyFile>
+    <OutputPath>$(OutputPath)Sdks\Microsoft.NET.Sdk\msbuildExtensions\Microsoft.NET.Build.Extensions\tools</OutputPath>
```

Comment:
Upon further thought, we should make this its own package. Doing it this way means the CLI build will have to do filtering on what it puts in the Microsoft.NET.Sdk folder. These are logically separate components that live together because they share code. They should be easy to trace through as distinct things in the CLI setup code.